src/controllers/checkins.ts

```diff
@@ -1,13 +1,14 @@
-import { CheckInInput, Event, EventInput } from './../models/checkin';
+import { CheckIn, CheckInInput, Event, EventInput } from './../models/checkin';
 import { Client } from '../models/client';
 import Express from "express";
 import { google } from "googleapis";
 import { serialize_rows, sheet_auth } from "../utils/sheets";
 import { v4 } from 'uuid';
 import { parse_mag_stripe } from '../utils/card';
+import { check_dup_student } from '../utils/checkInDup';
 
 const EVENT_SHEET_ID = "EVENTS";
-const CHECKIN_SHEET_ID = "CHECKINS";
+export const CHECKIN_SHEET_ID = "CHECKINS";
```

Comment:
Great. Eventually these should be moved into a util file of their own, it doesn't make sense to keep them here.

Reply:
Created a new task for this